.chglog/config-changelog.yml

```
@@ -0,0 +1,33 @@
+style: github
+template: CHANGELOG.tpl.md
+info:
+  title: CHANGELOG
+  repository_url: https://github.com/sysdiglabs/charts
+options:
+  commits:
+    filters:
+      Type:
+        - feat
+        - new
+        - fix
+        - perf
```

```suggestion
        - fix
```

very minor, but I believe we don't need `new` (when use `new` and when `feat` ?) and I can't think about `perf` use-cases.

Again, very minor comments, just trying to simplify the <what-should-I-use> dilemma when opening a PR 😄

---

(same consideration applies for the config-release.yml below)